src/selectors/locale.js

```diff
@@ -13,12 +13,14 @@ const supportedLocales = createSelector(
 	data => data.getIn(["supportedLocales"]) || Immutable.fromJS([]),
 );
 
-export const currentLocale = createSelector(
+export const currentLocaleOrDefault = createSelector(
```

Comment:
This will force all dependant apps to use the new selector name.

Reply:
I did not know how to name the new function to return the current locale even when the value was still null. I had preferred to rename the existing one. Update OMS & Analytics is not a big deal anyway.